crates/rbuilder/src/mev_boost/sign_payload.rs

```diff
@@ -148,11 +151,17 @@ pub fn sign_block_for_relay(
             block_hash: sealed_block.hash(),
             transactions: sealed_block
                 .body
+                .transactions
                 .iter()
-                .map(|tx| tx.envelope_encoded().to_vec().into())
+                .map(|tx| {
+                    let mut buf = Vec::new();
+                    tx.encode_2718(&mut buf);
+                    buf.into()
+                })
```

Comment:
Do we have a way to ensure the new encoding is correct?

Reply:
Not that I remember. We trust on reth tests.